src/safe-upgrade-tool/install.md

```diff
@@ -0,0 +1,80 @@
+---
+group: software-update-guide
+title: Install
+functional_areas:
+  - Upgrade
+---
+
+The Safe Upgrade Tool (SUT) is a command line (CLI) tool that checks a Magento instance against a specific version by analyzing all the non-Magento modules installed on it.
+
+## Workflow
+
+The following diagram shows the expected workflow when running the SUT:
+
+![SUT Diagram](img/mvp-diagram.png)
+
+### Who is the SUT for?
+
+The following use case describes the typical process for a Magento partner to upgrade a client's Magento instance:
```

Comment:
The described scenario is our "desired state", not a reality with current version of SUT.

At the moment SUT is capable of analyzing only a fraction of all extensibility scenarios. It means that system integrators must not exclusively rely on SUT output to forecast timeline and cost of the upgrade.

This may be important, depending on our goal with this documentation.

Reply:
I would understand that being an ALPHA, that section would be ideal scenario, which we can still keep, if @sgonzalezmangana thinks worth it, we could add a clarification for it